selfdrive/ui/qt/maps/map.cc

```diff
@@ -213,8 +213,7 @@ void MapWindow::updateState(const UIState &s) {
     // - API exception/no internet
     // - route response is empty
     // - any time navd is waiting for recompute_countdown
-    auto dest = coordinate_from_param("NavDestination");
-    routing_problem = !sm.valid("navInstruction") && dest.has_value();
+    routing_problem = !sm.valid("navInstruction") && coordinate_from_param("NavDestination");
```

Comment:
```suggestion
    routing_problem = !sm.valid("navInstruction") && coordinate_from_param("NavDestination").has_value();
```

I think this is clearer